packages/pretty-proptypes/src/usePropsExplorer/index.js

```diff
@@ -0,0 +1,31 @@
+import getPropTypes from '../getPropTypes';
+
+const getProps = props => {
+  if (props && props.component) {
+    return getPropTypes(props.component);
+  }
+  return null;
+};
+
+export default function usePropsExplorer(props) {
```

Comment:
Hey @davidfloegel: Love your work ❤️. Sorry for jumping like this. One question around naming here. Not sure if we really call this a hook or a util? 🤔

Reply:
Hey dude! Thanks :)

Yeah, I did like the idea of this being a hook but in the end I think it doesn't have to be. 

Daniel had a good point though about just exposing the getPropTypes function - so maybe that's more than enough